### DISCUSSION TITLE
Suggestions for enhanced functionality with "collection" models

### DISCUSSION THREAD

**Discussion by kmillerdevx3** [BlafKing/sd-civitai-browser-plus#185] [Ideas]
Some CivitAI creators put multiple versions of the same model in a collection. These are usually based around the same concept or base model.

I like to organize all of my downloaded models into subfolders. However, for collections of models like this, I have to download and organize each model individually. So these are some ideas for improving the CivitAI Browser Plus experience when downloading or updating models.

- "Sub folder" selection should apply to all versions in the collection
- "Download all versions" button instead of having to download each individually
- Checking boxes on multiple models and "Download all selected" should allow them to be organized into a subfolder (probably too complicated to try to let each be organized into its own subfolder, but a single subfolder for all selected should work.)
- Combining the previous two, "Download all selected versions" to download every version for all selected models, into a subfolder

An example collection: https://civitai.com/models/201305/school-dress-collection-by-stable-yogi

**Comment by BlafKing** [comment]
Sounds good!
I'll add this to the to-do list!

‎
> Checking boxes on multiple models and "Download all selected" should allow them to be organized into a subfolder

This is currently already possible if the selected models are of the same content type, so if you for example select a lot of LORA's then you can select a sub-folder where all the selected models will get downloaded to! :)

![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/9644716/82d08759-8507-4c23-b829-64266ee7f318)
